processor/probabilisticsamplerprocessor/probabilisticsampler_test.go

```diff
@@ -22,11 +22,11 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/idutils"
```

Comment:
The order is not good since this is local and collector is other

Reply:
I think goimports has that limitation. Just merge all the blocks and run again the goimports and will work